front/app/components/Form/Components/Controls/SentimentLinearScaleControl/utils.tsx

```diff
@@ -0,0 +1,85 @@
+import messages from '../messages';
+
+import Sentiment1Svg from './assets/sentiment_1.svg';
+import Sentiment2Svg from './assets/sentiment_2.svg';
+import Sentiment3Svg from './assets/sentiment_3.svg';
+import Sentiment4Svg from './assets/sentiment_4.svg';
+import Sentiment5Svg from './assets/sentiment_5.svg';
+
+export const getClassNameSentimentImage = (data, visualIndex) => {
+  if (data === visualIndex) {
+    return 'isSelected';
+  } else if (data) {
+    return 'anotherValueSelected';
+  }
+  return '';
+};
+
+export const handleKeyboardKeyChange = (event, value) => {
+  switch (event.key) {
+    case 'ArrowLeft':
+    case 'ArrowDown':
+      return Math.max(1, value - 1);
+    case 'ArrowRight':
+    case 'ArrowUp':
+      return Math.min(5, value + 1);
+    case 'Home':
+      return 1;
+    case 'End':
+      return 5;
+    case 'Enter':
+      return value;
+    default:
+      return;
+  }
+};
+
+export const getSentimentEmoji = (index: number) => {
+  switch (index) {
+    case 1:
+      return Sentiment1Svg;
+    case 2:
+      return Sentiment2Svg;
+    case 3:
+      return Sentiment3Svg;
+    case 4:
+      return Sentiment4Svg;
+    case 5:
+      return Sentiment5Svg;
+  }
+  return Sentiment1Svg;
+};
```

Comment:
Could simplify this to

```suggestion
const sentimentEmojis: Record<1 | 2 | 3 | 4 | 5, string> = {
  1: Sentiment1Svg,
  2: Sentiment2Svg,
  3: Sentiment3Svg,
  4: Sentiment4Svg,
  5: Sentiment5Svg,
};

export const getSentimentEmoji = (index: number) =>
  sentimentEmojis[index as keyof typeof sentimentEmojis];
```